### PR TITLE
Multiple image sources collection creation

### PR DESCRIPTION
**Pull request by tracivar**
- Updated schema for the multiple image sources creation yaml file. This should support file based, URL based and azure container imagesets. I have only tested file based ones. More thorough tests are required.
- Updated collection creation process to handle it
- Updated readme

### CODE REVIEW

**Review by ahaith — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -120,6 +120,37 @@ dataset_column: image_name
...
+file_config:
+    # Whether to recursively scan any directories. (optional)
+    recursive: True
+    # Path to the dataset file. (optional)
+    path: path/to/file/mydata.csv
+image_sources:
+    # source from file based imageset
+    - paths:
+        - a/directory/path
+      # source_name is a compusolory field
```
Comment (mariya-gfx) [inline README.md:141]: a typo in 'compulsory'

**Review by ahaith — COMMENTED** [review]
Need to add a version bump - this should be a minor version, so 1.5.0

File: README.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ image_sources:
     # source from file based imageset
     - paths:
         - a/directory/path
-      # source_name is a compusolory field
+      # source_name is a compulory field
```
Comment (mariya-gfx) [inline README.md:141]: a typo :D

Comment (tracivar) [reply README.md:141]: Oh gosh!

File: zeg/schemata/spec.yaml [hunk 3]
```diff
@@ -187,6 +187,46 @@ anyOf:
       - required:
         - imageset_type
         - container_name
+# multiple image sources collection
+- allOf:
+  - $ref: '#/definitions/collection'
+  - collection_version:
+    type: number
+    enum:
+    - 2
```
Comment (tarqu1n) [inline zeg/schemata/spec.yaml:196]: This means that collection_version: 1 is not valid which I think it should be.

File: zeg/schemata/spec.yaml [hunk 4]
```diff
@@ -187,6 +187,46 @@ anyOf:
       - required:
         - imageset_type
         - container_name
+# multiple image sources collection
+- allOf:
+  - $ref: '#/definitions/collection'
+  - collection_version:
```
Comment (tarqu1n) [inline zeg/schemata/spec.yaml:193]: should this just be `version`? to be consistent with cloud

Comment (tracivar) [reply zeg/schemata/spec.yaml:193]: This is the name we discussed to be and it's only used in the config file for user so it doesn't really matter.

**Pull request merged**